shared/src/util/errors.ts

```diff
@@ -21,6 +21,21 @@ export const asError = (value: unknown): Error => {
     return new Error(String(value))
 }
 
+const EAGGREGATEERROR = 'AggregateError'
```

Comment:
New convention since this PR was opened :)
```suggestion
const AGGREGATE_ERROR_NAME = 'AggregateError'
```